.github/workflows/ci.yml

```diff
@@ -254,17 +254,14 @@ jobs:
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
       - uses: ./.github/actions/build-zeebe
         id: build-zeebe
-      # Once we're on Go 1.18, use the official gorelease to do this
+      # Once we have breaking changes, this will break since no workaround atm.
+      # Give the official gorelease a try to do this
```

Comment:
offical go release of the go-apidiff pkg I assume? I think joelanford is quite trusty. From the comments I cannot tell why the official one was not used. But I guess its fine! 

I'm happy that there is progress here! 💪

Reply:
And now I know that I'm allowed to approve PR's here 🤯

Reply:
Thanks @Sijoma